src/model.cc

```diff
@@ -88,15 +88,15 @@ Model::SetScheduler(std::unique_ptr<Scheduler> scheduler)
 }
 
 Status
-Model::Init()
+Model::Init(const bool is_config_provided)
 {
   // If the model configuration has not been set, then look
   // whether the config file was explicitly provided.
   if (!set_model_config_) {
     const auto config_path = JoinPath({model_dir_, kModelConfigPbTxt});
     bool exists = false;
     RETURN_IF_ERROR(FileExists(config_path, &exists));
```

Comment:
Don't need this

Reply:
parsed_config already check the existence https://github.com/triton-inference-server/core/blob/7aa014ca08ade31f338158ddbd3aa83fb6a54726/src/model_repository_manager.cc#L1126-L1137

Reply:
Updated, thanks for the catch.